config-spring-property/src/main/java/org/stellar/anchor/server/config/PropertyResourceAppConfig.java

```diff
@@ -0,0 +1,17 @@
+package org.stellar.anchor.server.config;
+
+import java.util.List;
+import lombok.Data;
+import org.stellar.anchor.config.AppConfig;
+
+@Data
+public class PropertyResourceAppConfig implements AppConfig {
```

Comment:
❓ Can you briefly explain the relationship between these config classes, the properties files, and the application? How do they interact?

Reply:
I will be happy to explain on Monday. The document is a little intimidating to read.